bitvision/interface/bitvision-cli.js

```diff
@@ -0,0 +1,702 @@
+// GLOBALS
+"use strict";
+let fs = require("fs");
+let blessed = require("blessed");
+let contrib = require("blessed-contrib");
+let childProcess = require("child_process");
+let Gdax = require("gdax");
+let writeJsonFile = require("write-json-file");
+
+// ----------
+// MODULES
+// ----------
+
+let login = require("./login");
+let help = require("./help");
+
+// ----------
+// CONSTANTS
+// ----------
+
+let VERSION = "v0.1a"
+let MAX_HEADLINE_LENTH = 35;
+var helpOpenCloseCount = 0;
+// TODO: Figure out how to write to home directory.
+// const dotfilePath = "~/.bitvision";
+const dotfilePath = ".bitvision.json";
+var gdaxClient = new Gdax.PublicClient();
+
+// ---------
+// Bless up
+// ---------
+
+var screen = blessed.screen({
+  smartCSR: true,
+  title: "Bitvision",
+  cursor: {
+    artificial: true,
+    shape: "line",
+    blink: true,
+    color: "red"
+  }
+});
+
+// Quit
+screen.key(["escape", "C-c"], function(ch, key) {
+  return process.exit(0);
+});
+
+const log = (text) => {
+  logs.pushLine(text);
+  screen.render();
+}
+
+// -------------------------
+// DOTFILE RELATED FUNCTIONS
+// -------------------------
+
+function writeDotfile(config) {
```

Comment:
Do we have dotfiles? I thought we were going to store data in JSON files and settings/config in YAML files. A `.bitvision` file is cool though too.

Reply:
It's `~/.bitvision.json` atm but open to changing that.